src/components/NcSettingsSection/NcSettingsSection.vue

```diff
@@ -85,6 +86,16 @@ export default {
 			type: String,
 			default: '',
 		},
+		/**
+		 * Limit the width of the setting's content
+		 *
+		 * By default only the title and description have a limit, use this
+		 * property to also apply this to the rest of the content.
+		 */
+		limitWidth: {
+			type: Boolean,
+			default: false,
```

Comment:
Why not by default? :)

Reply:
Done. I was unsure if this was a good idea but after a night sleeping on it, it indeed make sense since nc7 is a breaking change release.